### PR TITLE
From feature/movie-form to develop !

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/credits/credits.component.ts [hunk 1]
```diff
@@ -15,12 +15,20 @@ export class CreditsComponent {
...
+  //   return this.form.get('salesCast');
+  // }
+
+  // get production() {
+  //   return this.form.get('production');
+  // }
+
+  // get production() {
+  //   return this.form.get('production');
+  // }
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/credits/credits.component.ts:33]: remove this if not used

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/evaluation/evaluation.component.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export class EvaluationComponent {
     this.dynTitle.setPageTitle('Marketplace Valuation', 'Title information')
   }
 
-  get salesInfo() {
-    return this.form.get('salesInfo');
-  }
+  // get salesInfo() {
+  //   return this.form.get('salesInfo');
+  // }
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/evaluation/evaluation.component.ts:20]: remove this if not used

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/main/main.component.html [hunk 3]
```diff
@@ -9,7 +9,7 @@ <h1 subtitle>Main Information</h1>
         <mat-card-title>Content Type</mat-card-title>
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/main/main.component.html:9]: I think the whole old movie tunnel can be removed

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.component.ts [hunk 4]
```diff
@@ -71,6 +71,13 @@ const steps: TunnelStep[] = [{
 })
 export class MovieTunnelComponent implements TunnelRoot, OnInit {
   steps = steps;
+
+
+  //////////////////////////////////////
+  // EVERYTHING BELOW WILL BE REMOVED //
+  //////////////////////////////////////
+
+
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.component.ts:80]: lets remove it then !

File: styles/components/mat-list.scss [hunk 5]
```diff
@@ -7,6 +7,11 @@
   $background: map-get($theme, background);
   $foreground: map-get($theme , foreground);
 
+  // use mainly for the sidenav in movie tunnel, I can't override this in the component, it has to be at the style root.
+  .mat-list-base .mat-list-item .mat-list-item-content {
+    padding: 0;
+  }
+
```
Comment (phpgeekfr) [inline styles/components/mat-list.scss:14]: I think this is causing the missing padding in the right top corner (profile menu)

File: angular.json [hunk 6]
```diff
@@ -561,7 +562,10 @@
             "scripts": ["node_modules/apexcharts/dist/apexcharts.min.js"]
           },
           "configurations": {
-            "dev": {},
+            "dev": {
```
[the remark this answers is not in the document]
Comment (phpgeekfr) [reply angular.json:565]: Me neither, we need to remove it ?

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**